styles/styles.css

```diff
@@ -600,9 +600,7 @@ main .section.highlight {
 }
 
 .grid-item--50 {
-  @media (min-width: 22.5rem) {
-    grid-column: span 4;
-  }
+  grid-column: span 8;
```

Comment:
I don't think my brain realized we are doing an 8-column grid instead of a 4-column grid when I wrote these up, or that changed since. **Can you either change the base grid on line 574 to be:** 

```css
grid-template-columns: repeat(4, [col] 1fr);
```

**or change the other classes to account for 8 columns**
`.grid-item--66`, `grid-item--33` small to be span 8

then this one needs to be at 50% so: 

```css
.grid-item--30 {
  @media (min-width: 22.5rem) {
    grid-column: span 4;
  }
```